src/lib.rs

```diff
@@ -646,18 +646,19 @@ pub type Ixs = isize;
 /// are supported (scalar can be on either the left or right side, but
 /// `ScalarOperand` docs has the detailed condtions).
 ///
-/// - `&A @ K` or `K @ &A` which produces a new `Array`
-/// - `B @ K` or `K @ B` which consumes `B`, updates it with the result and returns it
-/// - `C @= K` which performs an arithmetic operation in place
+/// - `&A @ K`, `V @ K`, `K @ &A`, or `K @ V` which produces a new `Array`
+/// - `O @ K` or `K @ O` which consumes `O`, updates it with the result and returns it
+/// - `M @= K` which performs an arithmetic operation in place
 ///
 /// ### Unary Operators
 ///
-/// Let `A` be an array or view of any kind. Let `B` be an array with owned
-/// storage (either `Array` or `ArcArray`). The following operands are supported
-/// for an arbitrary unary operator denoted by `@` (it can be `-` or `!`).
+/// Let `A` be an array or view of any kind. Let `O` be an array with owned
+/// storage (either `Array` or `ArcArray`). Let `V` be an `ArrayView`. The
+/// following operands are supported for an arbitrary unary operator denoted by
+/// `@` (it can be `-` or `!`).
 ///
-/// - `@&A` which produces a new `Array`
-/// - `@B` which consumes `B`, updates it with the result, and returns it
+/// - `@&A` or `@V` which produces a new `Array`
+/// - `@O` which consumes `O`, updates it with the result, and returns it
```

Comment:
Thanks for updating this, even if it's a draft PR or whatever, it makes it easy to discuss changes :)